pkg/operators/kubeipresolver/kubeipresolver.go

```diff
@@ -226,72 +104,51 @@ func (m *KubeIPResolverInstance) PostGadgetRun() error {
 }
 
 func (m *KubeIPResolverInstance) enrich(ev any) {
-	additionalInfo, _ := ev.(KubeNetworkInformation)
```

Comment:
BTW, it's unrelated to this PR, but could you please update `KubeIPResolverInstance.Name()`?